tests/services/test_file_mover.py

```
@@ -0,0 +1,258 @@
+def test_copy(mock_copy_with_progress, source_and_dest):
+    """Test that _copy calls _copy_with_progress."""
+    source, dest = source_and_dest
+    success, error = _copy(source, dest, None)
+
+    assert success is True
+    assert error is None
+    mock_copy_with_progress.assert_called_once_with(source, dest, None)
+
+
+@patch("shutil.copy2", side_effect=Exception("Disk full"))
+def test_copy_exception(mock_copy2, source_and_dest):
```

Ruff (0.14.14)

[warning] Unused function argument: `mock_copy2`

(ARG001)